presto-main/src/main/java/com/facebook/presto/operator/JoinHashSupplier.java

```diff
@@ -109,4 +110,18 @@ public JoinHash get()
                     return links.create(searchFunctions);
                 }));
     }
+
+    /**
+     * @return ONLY FOR TESTING -- a roughly correct understanding of how much memory this object uses.
```

Comment:
The problem with the comments like this - that people not always read them. Especially if they rely on IDEs with great index, such as Intellij =)

Reply:
Sure -- but what's the alternative? Renaming the method name?